config/webpack.base.js

```diff
@@ -17,6 +16,18 @@ module.exports = {
             'redux-thunk',
         ],
     },
+    optimization: {
```

Comment:
Do we need to specify our venor code splitting anymore? My understanding is webpack 4 now automagically splits out vendors as needed by default.

Reply:
I tried without an explicit vendor chunk definition and it put all together in the client bundle. Maybe it has a single configuration param to do that and I'm not aware.